tests/www/companies_views/test_job_description_views.py

```diff
@@ -100,16 +100,17 @@ def test_job_application_list_response_content(self):
         # 5.  SAVEPOINT
         # 6.  SELECT COUNT companies_jobdescription
         # 7.  SELECT COUNT companies_companymembership
-        # 8.  SELECT companies_siaeconvention (menu checks for financial annexes)
-        # 9.  SELECT EXISTS users_user (menu checks for active admin)
-        # 10. SELECT companies_jobdescription
-        # 11. SELECT jobs_appellation
-        # 12. SELECT jobs_rome
-        # 13. RELEASE SAVEPOINT
-        # 14. SAVEPOINT
-        # 15. UPDATE django_session
-        # 16. RELEASE SAVEPOINT
-        with self.assertNumQueries(16):
+        # 8.  SELECT EXISTS users_user (convention_can_be_accessed_by())
+        # 9.  SELECT companies_siaeconvention (menu checks for financial annexes)
+        # 10.  SELECT EXISTS users_user (menu checks for active admin)
```

Comment:
Vivement `assertNumQueriesSnapshot` :grin: 
```suggestion
        # 10. SELECT EXISTS users_user (menu checks for active admin)
```

Reply:
Oui ! :eyes: @xavfernandez